src/axom/slic/docs/sphinx/index.rst

```diff
@@ -0,0 +1,9 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
```

Comment:
@gzagaris , I know you have the slic docs PR in progress.  This is a placeholder.

Reply:
Thanks. I'll rebase my branches based on this.